tests/framework/utils_cpuid.py

```diff
@@ -10,6 +10,14 @@
 from framework.utils import run_cmd
 from framework.utils_imdsv2 import imdsv2_get
 
+INTEL_CPU_DICT = {
+    "Intel(R) Xeon(R) Platinum 8175M CPU": "INTEL_SKYLAKE",
+    "Intel(R) Xeon(R) Platinum 8259CL CPU": "INTEL_CASCADELAKE",
+    "Intel(R) Xeon(R) Platinum 8375C CPU": "INTEL_ICELAKE",
+}
+AMD_CPU_DICT = {
+    "AMD EPYC 7R13": "AMD_MILAN",
+}
 ARM_CPU_DICT = {"0xd0c": "ARM_NEOVERSE_N1", "0xd40": "ARM_NEOVERSE_V1"}
```

Comment:
Maybe this could get unified in a dict like `CPU_DICT[arch, name]`? So we only have to look up in one dict.

Reply:
Sounds cool! done.